### ISSUE TITLE
Mailers: an outdated assignment reading numbers

### ISSUE THREAD

**Issue by MahdiRox** [TheOdinProject/curriculum#29260]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

in the assignment the sections that are ordered to be read are outdated and don't match the what's in the rails website anymore so they should be updated <3

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/ruby-on-rails-mailers

**Comment by JoshDevHub** [comment]
Thank you for making this issue @MahdiRox -- you're absolutely correct that this should be updated.

### Acceptance Criteria
- [ ] In the assignment section of the [Rails Mailers lesson](https://www.theodinproject.com/lessons/ruby-on-rails-mailers), update the section reading to match with the new layout of the information.
    - The first item can remain unchanged.
    - The second item needs to point to sections 6 and 7 of the mailers guide.
    - The third item needs to point to section 8.2 of the mailers guide.
    - The fourth item needs to sections 9 and 10 of the mailers guide.
    
Since the OP indicated they do not wish to work on this themselves, I'll open this up for any contributors who wish to submit a PR. Comment below, and a maintainer will assign you.

**Comment by MahdiRox** [comment]
Yes im down to work on it

**Comment by JoshDevHub** [comment]
Okay @MahdiRox I'll assign you.

**Comment by Yateen00** [comment]
Hey @MahdiRox, I saw that you were assigned to this issue a while ago. I recently came across this section too, and it looks like a small fix. Just wanted to check if you’re still working on it. If not, I wouldn’t mind taking a look. Let me know. Thanks!

**Comment by MahdiRox** [comment]
Yes sure lately i stopped studying for some personal reasons and im sorry for it if you wonna take and fix that could be amazing and i repeat im sorry

**Comment by Yateen00** [comment]
No problem at all, @MahdiRox! Hope things get better for you. @JoshDevHub , would it be possible to assign me to this issue?

**Comment by JoshDevHub** [comment]
Yes, I will assign it to you. Thank you for volunteering @Yateen00
